docs/reference/sql/endpoints/version-compat.asciidoc

```diff
@@ -0,0 +1,43 @@
+Your driver must be compatible with your {es} server version.
+
+IMPORTANT: The driver version cannot be newer than the {es} server version.
+For example, {version} drivers are not compatible with 7.10.0 servers.
+
+[options="header",cols="<1,<1,<1"]
+|====
+| {es} server version
+| Compatible driver versions
+| Examples
+
+| 7.7.1–{version}
+a| 
+* The same version.
+* Any earlier version from the server's major version. If the server's major
```

Comment:
Thanks, @jrodewig for updating it. It's even more succinct and clearer now, but to @astefan's point, this format will require updating. But we should be good for a good while from now.

Reply:
Thanks @bpintea. Yes. This will require an update for the 8.0 release and any future major releases. I added a comment as a reminder for the docs team.